MvvmCross.Forms/Platforms/Android/Core/MvxFormsAndroidSetup.cs

```diff
@@ -43,6 +54,16 @@ protected override void InitializeIoC()
         {
             base.InitializeIoC();
             Mvx.RegisterSingleton<IMvxFormsSetup>(this);
+
+            Mvx.LazyConstructAndRegisterSingleton<IMvxViewPresenter, MvxFormsAndroidViewPresenter>();
+            Mvx.LazyConstructAndRegisterSingleton<IMvxFormsSetupHelper, MvxFormsSetupHelper>();
+            Mvx.Resolve<IMvxFormsSetupHelper>().InitializeIoC();
```

Comment:
`LazyConstructAndRegisterSingleton<IMvxFormsSetupHelper, MvxFormsSetupHelper>` here doesn't make much sense if we're going to call `Resolve<IMvxFormsSetupHelper>` in the next line 🤔

Reply:
Agreed, it was more for consistency than anything - I'll adjust this